### DISCUSSION TITLE
New Camera - Yi 1080p PTZ outdoor Camera- Support for QFUS -

### DISCUSSION THREAD

**Discussion by ABrauser** [roleoroleo/yi-hack-Allwinner-v2#173] [General]
Hello Together,

is it possible to customize the firmware to support the yi 1080p PTZ outdoor Camera or is it all ready compatible? This model is quite new I guess. and I bought it via Ali Express. it is fully integrated with the Yi Home App, but I was not able to find a custom firmware to use the rtsp streams. Regarding some Discord channels some of the developer mentioned that your firmware is the closest one that is compatible with the camera. Is the true or does it require some development time to get it working? Firmware Version is 9.0.45.00_202101251914 and the ID Beginns with "QFUSXXXXXXX"

**Comment by roleoroleo** [comment]
The cam is actually not compatible.
To check if Allwinner-v2 is compatible with this cam I need a dump of the flash.
Check the wiki.

**Reply by ABrauser** [reply]
Hi, i have placed the config.sh and factory_test.sh in the root of the sd card. after several reboots and waitings for 3-4 Minutes no "/backup" folder is created. i also reset the camera to make sure the script will run. Looks like the script is not running this way. I am doing something wrong? the only files that were created is a "log" folder wiht a "log_first_login.tar.gz" and "log_wifi_connected.tar.gz" inside there are some file about the file structure of the camera. are those maybe usefull? Also available under (https://1drv.ms/u/s!AkEbjqHIUf-og-YJSc8r50-TFXb8_w?e=3eqK7U) I have uploaded it to my onedrive. i also noticed an update of the firmware from 9.0.45.00_202101251914 to 9.0.45.02_202105081636 but i didnt performed it yet.
[df.txt](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/6540741/df.txt)
[file.txt](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/6540742/file.txt)
[lsmod.txt](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/6540743/lsmod.txt)
[ps1.txt](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/6540744/ps1.txt)

**Comment by roleoroleo** [comment]
Probably this isn't an Allwinner-v2.
Check if the backup procedure for Allwinner works:
https://github.com/roleoroleo/yi-hack-Allwinner/wiki/Dump-your-backup-firmware-(SD-card)

**Reply by ABrauser** [reply]
Tried it with the other Firmware Dump, no backup folder was created aswell. Is it possible to check if any commands are executed? The camera also starts to rotate when plug in, also the Light is turning on and off after 5 seconds. lts doing like a startup checks. Can this maybe interfear with the creation of the backup?

**Comment by roleoroleo** [comment]
Checking lsmod and ps it seems an Allwinner-v2 cam.

> Hi, i have placed the config.sh and factory_test.sh in the root of the sd card.

Check the path, you have to copy the Factory folder, not the files in the root.

If it doesn't work, maybe you sd card is not supported.
Try another card or try to reformat it.

**Reply by ABrauser** [reply]
Just used a 32GB sd card. (Windows also was able to format to FAT32). With the 64GB i had to use external tools. Also copied the "Factory" folder. Now it worked. Here are the Backup Files. You can download the Backup.7z from my one drive. https://1drv.ms/u/s!AkEbjqHIUf-og-YhzI257p7fqDting?e=ofvzQj let me know if you need some more information or if i can help you with the process or you need me to test the firmware! Ty for your assistance! ! :-)

**Reply by roleoroleo** [reply]
Your model is r40ga.
At the moment it's not supported.
But I could try to create a beta.

**Reply by roleoroleo** [reply]
Check the issue you already opened:
https://github.com/roleoroleo/yi-hack-Allwinner-v2/issues/161
I will update this issue.

**Reply by michalszym** [reply]
I also have this camera and I would like to know if the beta version of yi-hack was made.

**Reply by roleoroleo** [reply]
Now it's released.
